### PR TITLE
Fixing schema.sql

### PR DESCRIPTION
**Pull request by indiah444**
- Changed the double quotes used around the seed data to single quotes in line with SQL constraints.
- Updated the order of `DROP TABLE IF EXISTS` commands to make sure tables were dropped in correct order.

### CODE REVIEW

File: database/schema.sql [hunk 1]
```diff
@@ -1,7 +1,7 @@
-DROP TABLE IF EXISTS source;
+DROP TABLE IF EXISTS article_topic_assignment;
```
Comment (YakMan101) [inline database/schema.sql:1]: Good reordering to avoid error related to foreign keys

**Review by YakMan101 — CHANGES_REQUESTED** [review]
I think each table should have constraints according the the combination of columns that should be unique. 
e.g for `article` table, maybe not have `article_title` as unique but have combination of:
`(article_title, source_id, date_published)` 
as a unique. incase two different articles happen to have the same title.

File: database/schema.sql [hunk 2]
```diff
@@ -14,12 +14,13 @@ CREATE TABLE source (
 
 CREATE TABLE article (
     article_id BIGINT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
-    article_title VARCHAR(500) NOT NULL UNIQUE,
+    article_title VARCHAR(500) NOT NULL,
     polarity_score FLOAT NOT NULL,
```
Comment (megansam5) [inline database/schema.sql:18]: sorry to be annoying, but I think we are now also keeping article_content (probably text as it may be large)

**Review by megansam5 — CHANGES_REQUESTED** [review]

**Review by YakMan101 — APPROVED** [review]

File: database/schema.sql [hunk 3]
```diff
@@ -14,12 +14,14 @@ CREATE TABLE source (
 
 CREATE TABLE article (
     article_id BIGINT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
-    article_title VARCHAR(500) NOT NULL UNIQUE,
+    article_title VARCHAR(500) NOT NULL,
+    article_content TEXT NOT NULL,
```
Comment (megansam5) [inline database/schema.sql:18]: Thank you for adding the content!

**Review by megansam5 — APPROVED** [review]
All looks good!

**Pull request merged**